dace/libraries/blas/nodes/axpy.py

```diff
@@ -130,24 +141,27 @@ def make_sdfg(dtype, veclen, n, a, buffer_size_x, buffer_size_y,
             'axpy_task', ['x_con', 'y_con'], ['z_con'],
             'z_con = {} * x_con + y_con'.format(a))
 
+        access_vol = '0' if streaming else 'i'
```

Comment:
Isn't this an index, not a volume? The `dace.Memlet.simple(x_in.data, "0")` API is actually deprecated in favor of `dace.Memlet(f"{x_in.data}[{access_vol}]")`

Reply:
Fixed and updated to new API